### PR TITLE
[2203] Revert "Merge pull request #141 from yanqiongchen/backup-admin-update"

### PR DESCRIPTION
**Pull request by yanqiongchen**
Revert 2203 backup admin powershell update as our payload is reverted.

### CODE REVIEW

File: src/Azs.Backup.Admin/test/Utils.ps1 [hunk 1]
```diff
@@ -11,8 +11,6 @@ function setupEnv() {
     # as default. You could change them if needed.
```
Comment (TheOnlyWei) [inline src/Azs.Backup.Admin/test/Utils.ps1:11]: Is this file used at all? I don't see any references to this file.

**Review by TheOnlyWei — APPROVED** [review]

File: src/Azs.Backup.Admin/test/Utils.ps1 (hunk 1 above)
Comment (yanqiongchen) [reply src/Azs.Backup.Admin/test/Utils.ps1:11]: It is used to set the env file, we will keep this file.

**Pull request merged**